lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -53,7 +53,6 @@
         <hr class="divider divider-lv1" />
 
         <!-- Discussion Topics. -->
-        <button class="toggle-cohort-management-discussions" data-href="#cohort-discussions-management"><%- gettext('Specify whether discussion topics are divided by cohort') %></button>
         <div class="cohort-discussions-nav is-hidden" id="cohort-discussions-management" tabindex="-1">
```

Comment:
You need to delete all of this stuff from the cohorts instructor dashboard section.